callautomation-connect-rooms initial changes. (#301)
Azure-Samples/communication-services-javascript-quickstarts: callautomation-connect-rooms-quickstart/README.md

|page_type|languages|products
|---|---|---|
|sample|<table><tr><td>Typescript</tr></td></table>|<table><tr><td>azure</td><td>azure-communication-services</td></tr></table>|

> # Call Automation - Quick Start Sample

Reviewer: ```suggestion
# Connect to a Rooms call using Call Automation SDK

> In this quickstart, we cover how you can use Call Automation SDK to connect to active room call with connect endpoint. Creating room with users and enabling pstn dialout to add pstn participant.

Reviewer: ```suggestion
In this sample, we cover how you can use Call Automation SDK to connect to an active Azure Communication Services Rooms call with a connect endpoint. This involves creating an ACS Rooms with users and enabling PSTN dialout to add PSTN participant.
```

2. `ACS_RESOURCE_PHONE_NUMBER`: Phone number associated with the Azure Communication Service resource. For e.g. "+1425XXXAAAA"
3. `TARGET_PHONE_NUMBER`: Target phone number to add in the call. For e.g. "+1425XXXAAAA"
4. `CALLBACK_URI`: Base url of the app. (For local development replace the dev tunnel url)

### Run app locally

1. Open a new Powershell window, cd into the `callautomation-connect-rooms-quickstart` folder and run `npm run dev`
2. Browser should pop up with the below page. If not navigate it to `http://localhost:8080/`
3. To connect rooms call, click on the `Connect a call!` button or make a Http get request to https://<CALLBACK_URI>/connectCall

### Creating and connecting to room call.

1. ![create room with user](./data/createRoom.png)
2. Open two tabs for Presenter and attendee  ![calling room quickstart](./data/callingRoomQuickstart.png) 
3. Copy tokens for presenter and attendee from ![tokens](./data/tokens.png)
4. Initialize call agent with tokens for both presenter and attendee.
5. Take room id ![room id](./data/roomId.png) and initiate rooms call for both users. ![join room call](./data/joinRoomCall.png)

> 6. Connect room call with callautomation connect call endpoint. ![connect room call](./data/connectCall.png)

Reviewer: ```suggestion
6. Connect room call with call automation connect call endpoint. ![connect room call](./data/connectCall.png)
```
